src/barkhaPJun20/PinValidation.java

```diff
@@ -0,0 +1,74 @@
+/*WAP to validate Pin number.
+
+Pin number is valid if it follows below rules :
+1) ATM pin length should be 4.
+2) All characters must be digit.
+3) Either all digits are in ascending order or in descending order. 
+4) Same digit should not be repeated in a pin number.
+
+Input : 9654 -> Valid pin
+Input : 3578 -> Valid pin
+Input : 97A2 -> Invalid pin
+Input : 9553 -> Invalid pin
+Input : 3687 -> Invalid pin
+Input : 98765 -> Invalid pin
+Input : 76$2 -> Invalid pin
+
+It would be great if you think solution by below approach :
+-> One with collections.
+-> Without using collections. */
+package barkhaPJun20;
+
+public class PinValidation {
+
+	boolean isDigit(String pin) {
+		for (int index = 0; index < pin.length(); index++) {
+			if (Character.isDigit(pin.charAt(index)))
```

Comment:
you should use isLetter() instead of isDigit and second point return false if any char is letter